astropy/units/quantity.py

```diff
@@ -1173,6 +1174,9 @@ def __bool__(self):
         """Quantities should always be treated as non-False; there is too much
         potential for ambiguity otherwise.
         """
+        warnings.warn('The truth value of a Quantity is amiguous. '
```

Comment:
Small typo here and in the test: ambiguous.

Reply:
Fixed.